src/components/EndangeredSpecies/BandsSex.jsx

```diff
@@ -0,0 +1,83 @@
+import { React } from 'react';
+import PropTypes from 'prop-types';
+import { InfoIcon } from '@chakra-ui/icons';
+import {
+  Text,
+  VStack,
+  HStack,
+  Input,
+  Tooltip,
+  Select,
+  Heading,
+  FormControl,
+  FormLabel,
+} from '@chakra-ui/react';
+import footNotes from './FootNotes';
+
+function BandsSex({ totalAdults, totalFledges }) {
```

Comment:
For consistency, please use arrow functions ex: `const BandsSex = ({ totalAdults, totalFledges }) => {`